src/pac-file-tester.ts

```diff
@@ -92,7 +89,11 @@ const vmContext = (ip: string) => {
   }
 
   const isPlainHostName = (host: string) => {
-    return (host.search('\\\\.') == -1);
+    return host.indexOf(".") === -1;
+  }
+
+  const dnsDomainIs = (host: string, domain: string) => {
+    return host.endsWith(domain);
```

Comment:
Added this one as it was in a PAC file I was testing:
https://findproxyforurl.com/pac-functions/

Reply:
This is an odd one. checking if `some.domain.tld` ends `domain.tld` is not the same as the way findproxyforurl.com explains it, however it does appear to be the way [browsers implemented it](https://bugs.chromium.org/p/chromium/issues/detail?id=299649#c4).

Little odd but I'd rather have it work this way so it matches browsers.

_this comment is here more for a record incase I get bug reports in the future_